zebra/zebra_evpn_mac.c

```diff
@@ -972,6 +980,7 @@ zebra_mac_t *zebra_evpn_mac_add(zebra_evpn_t *zevpn, struct ethaddr *macaddr)
 	mac->neigh_list = list_new();
 	mac->neigh_list->cmp = neigh_list_cmp;
 
+	mac->uptime = monotime(NULL);
```

Comment:
Is the goal to set the uptime when a local mac is added? or on all mobility events? local->remote, remote->local, local1->local2, remote1->remote2)? If latter there are a few other places were you would need to set it.